src/app/AlwaysOn.Shared/TelemetryExtensions/AppInsightsRequestHandler.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.Azure.Cosmos;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace AlwaysOn.Shared.TelemetryExtensions
+{
+    public class AppInsightsCosmosRequestHandler : RequestHandler
```

Comment:
how about a small comment on class level what this is doing?

Reply:
you added it on the constructor now. Shouldnt it be on the class?

Reply:
..... 🤦